applications/the101/android/src/main/kotlin/io/tagd/the101/android/SampleAppLoadingStateHandler.kt

```diff
@@ -41,9 +24,7 @@ class SampleAppLoadingStateHandler(application: TagdApplication) :
         }
 
         compute (delay = 3000L) {
-            present {
-                onComplete(state)
-            }
+            onComplete(APP_SPECIFIC_LOADING_STATE_HEAVY_BG_WORK)
```

Comment:
Just some code enhancement (not an immediate requirement):

there can be further scope to remove this explicit call for `onComplete(stepName)`.
Since the `stepName` was already passed while registering,

Reply:
will discuss